konduit-serving-models/konduit-serving-tensorflow/src/test/java/ai/konduit/serving/models/tensorflow/TestTensorFlowStep.java

```diff
@@ -102,16 +102,16 @@ public void testFrozenModelGraph() throws Exception {
 
         GraphStep i2n = camera.then("image2NDArray", new ImageToNDArrayStep()
                 .config(c)
-                .keys(Arrays.asList("image"))
-                .outputNames(Arrays.asList("image_tensor")) //TODO varargs builder method
+                .keys("image")
+                .outputNames("image_tensor") //TODO varargs builder method
```

Comment:
We should go through and remove all of these `//TODO varargs builder method` comments now :)